src/olympia/api/tests/test_fields.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+from django.core.exceptions import ValidationError
+
+from mock import Mock
+from nose.tools import eq_
+from rest_framework.request import Request
+from rest_framework.serializers import Serializer
+from rest_framework.test import APIRequestFactory
+
+from olympia.addons.models import Addon
+from olympia.api.fields import (ESTranslationSerializerField,
```

Comment:
Can we indent this like so (I switched to that format most of the time during the module-restructure and we discussed that it makes sense):

```python
from olympia.api.fields import (
    ESTranslationSerializerField, TranslationSerializerField)
```